### ISSUE TITLE
Add undo-fu for evil mode

### ISSUE THREAD

**Issue by jacksonludwig** [daviwil/emacs-from-scratch#29]
I may be wrong, but according to evil mode's instructions, an undo package should be installed to correctly simulate vim's undo (undo-fu, undo-tree, or emacs 28's undo-redo). Just a ``(use-package undo-fu)`` is enough I believe.

**Comment by daviwil** [comment]
Yep, I need to add a note for that in the evil-mode video, thanks for bringing it up!
